From 1e034561de36b0eda6356b15b5433bf8571d4067 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Sun, 30 Oct 2022 11:59:10 +0900
Subject: [PATCH] test: support non-summer time

Follow-up for 759ed0a2533da8840dea315d07f92e6bb0272cdd.

(cherry picked from commit 59ab79a73d030a49bfdffd85897b6b30a2b132c5)

Related: #2155516
---
 test/units/testsuite-45.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/units/testsuite-45.sh b/test/units/testsuite-45.sh
index 75e07bc09a..38017a43ea 100755
--- a/test/units/testsuite-45.sh
+++ b/test/units/testsuite-45.sh
@@ -39,7 +39,7 @@ test_timezone() {
     if [[ -f /etc/timezone ]]; then
         assert_eq "$(cat /etc/timezone)" "Europe/Kiev"
     fi
-    assert_in "Time zone: Europe/Kiev \(EEST, \+0[0-9]00\)" "$(timedatectl)"
+    assert_in "Time zone: Europe/Kiev \(EES*T, \+0[0-9]00\)" "$(timedatectl)"
 
     if [[ -n "$ORIG_TZ" ]]; then
         echo 'reset timezone to original'
